bool reset_phase_position # True to use 'phase_position', False to ignore
float32 phase_position    # phase of grating (radians)
float32 phase_velocity    # velocity of grating phase (radians per second)
float32 wavelength        # spatial wavelength of grating (radians)
float32 contrast          # Michelson contrast of grating
float32 orientation       # orientation of grating (radians)
float32 mask_position     # position of mask center (radians)
float32 mask_width        # width of mask (radians)
float32 mask_intensity    # intensity of mask (radians)
